app/config/settings_prod.py

```diff
@@ -77,6 +80,7 @@
 
 ROOT_URLCONF = 'config.urls'
 API_BASE = 'api/stac/v0.9'
+LOGIN_URL = f"/{API_BASE}/login/"
```

Comment:
the default from django is `/api/stac/admin/login` so is this additional or replacement? Can we do with just one?

Reply:
One is enough, you're right.
As discussed, now there's no more login button on the nav-bar and the only login available is `/api/stac/admin/login/`.